2025-09-21 11:27:02.637 WARNING (Recorder) [homeassistant.components.recorder.util] The system could not validate that the sqlite3 database at //config/home-assistant_v2.db was shutdown cleanly
2025-09-21 11:27:02.745 WARNING (Recorder) [homeassistant.components.recorder.util] Ended unfinished session (id=29 from 2025-07-30 20:41:18.498608)
2025-09-21 11:27:06.480 WARNING (ImportExecutor_0) [py.warnings] /usr/local/lib/python3.13/site-packages/google/__init__.py:2: UserWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html. The pkg_resources package is slated for removal as early as 2025-11-30. Refrain from using this package or pin to Setuptools<81.
  __import__('pkg_resources').declare_namespace(__name__)

2025-09-21 11:27:08.027 WARNING (ImportExecutor_0) [py.warnings] /usr/local/lib/python3.13/site-packages/google/protobuf/runtime_version.py:98: UserWarning: Protobuf gencode version 5.29.5 is exactly one major version older than the runtime version 6.31.1 at api.proto. Please update the gencode to avoid compatibility violations in the next runtime release.
  warnings.warn(

2025-09-21 11:27:08.029 WARNING (ImportExecutor_0) [py.warnings] /usr/local/lib/python3.13/site-packages/google/protobuf/runtime_version.py:98: UserWarning: Protobuf gencode version 5.29.5 is exactly one major version older than the runtime version 6.31.1 at api_options.proto. Please update the gencode to avoid compatibility violations in the next runtime release.
  warnings.warn(

2025-09-21 11:27:10.328 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensornurse_call_button already exists - ignoring binary_sensor.d1_mqqt2_nurse_call_button
2025-09-21 11:27:10.329 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensornurse_call_button already exists - ignoring binary_sensor.d1_mqqt2_nurse_call_button
2025-09-21 11:27:10.331 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorcode_blue_button already exists - ignoring binary_sensor.d1_minipro_code_blue_button
2025-09-21 11:27:10.336 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensornurse_call_button already exists - ignoring binary_sensor.d1_mqqt2_nurse_call_button
2025-09-21 11:27:10.338 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorcode_blue_button already exists - ignoring binary_sensor.d1_minipro_code_blue_button
2025-09-21 11:27:10.341 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorpir_motion_sensor already exists - ignoring binary_sensor.lolinesp32_pir_motion_sensor
2025-09-21 11:27:10.344 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensornurse_call_button already exists - ignoring binary_sensor.d1_mqqt2_nurse_call_button
2025-09-21 11:27:10.347 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorcode_blue_button already exists - ignoring binary_sensor.d1_minipro_code_blue_button
2025-09-21 11:27:10.355 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorpir_motion_sensor already exists - ignoring binary_sensor.lolinesp32_pir_motion_sensor
2025-09-21 11:27:10.357 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensornurse_call_button already exists - ignoring binary_sensor.d1_mqqt2_nurse_call_button
2025-09-21 11:27:10.360 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorcode_blue_button already exists - ignoring binary_sensor.d1_minipro_code_blue_button
2025-09-21 11:27:10.362 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorpir_motion_sensor already exists - ignoring binary_sensor.lolinesp32_pir_motion_sensor
2025-09-21 11:27:10.374 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorcode_blue_button already exists - ignoring binary_sensor.d1_minipro_code_blue_button
2025-09-21 11:27:10.390 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorroom_light_button already exists - ignoring binary_sensor.bedsidev2_room_light_button
2025-09-21 11:27:10.395 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorcode_blue_button already exists - ignoring binary_sensor.d1_minipro_code_blue_button
2025-09-21 11:27:10.398 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorroom_light_button already exists - ignoring binary_sensor.bedsidev2_room_light_button
2025-09-21 11:27:10.399 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorclean_theatre_button already exists - ignoring binary_sensor.theatre_module_clean_theatre_button
2025-09-21 11:27:10.406 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorcode_blue_button already exists - ignoring binary_sensor.d1_minipro_code_blue_button
2025-09-21 11:27:10.408 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorroom_light_button already exists - ignoring binary_sensor.bedsidev2_room_light_button
2025-09-21 11:27:10.409 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorclean_theatre_button already exists - ignoring binary_sensor.theatre_module_clean_theatre_button
2025-09-21 11:27:10.411 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorcode_blue_button already exists - ignoring binary_sensor.d1_minipro_code_blue_button
2025-09-21 11:27:10.413 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorroom_light_button already exists - ignoring binary_sensor.bedsidev2_room_light_button
2025-09-21 11:27:10.416 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorclean_theatre_button already exists - ignoring binary_sensor.theatre_module_clean_theatre_button
2025-09-21 11:27:10.426 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorcode_blue_button already exists - ignoring binary_sensor.d1_minipro_code_blue_button
2025-09-21 11:27:10.427 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorclean_theatre_button already exists - ignoring binary_sensor.theatre_module_clean_theatre_button
2025-09-21 11:27:10.430 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorroom_light_button already exists - ignoring binary_sensor.bedsidev2_room_light_button
2025-09-21 11:27:10.435 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorcode_blue_button already exists - ignoring binary_sensor.d1_minipro_code_blue_button
2025-09-21 11:27:10.456 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorin_use_button already exists - ignoring binary_sensor.theatre_module_esp32_in_use_button
2025-09-21 11:27:10.465 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorcleaning_button already exists - ignoring binary_sensor.fire4_cleaning_button
2025-09-21 11:27:10.497 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorcode_blue_button already exists - ignoring binary_sensor.d1_minipro_code_blue_button
2025-09-21 11:27:10.498 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorin_use_button already exists - ignoring binary_sensor.theatre_module_esp32_in_use_button
2025-09-21 11:27:10.500 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform mqtt does not generate unique IDs. ID ESPbinary_sensorlight_button already exists - ignoring binary_sensor.fire4_light_button
2025-09-21 11:27:10.538 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.545 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.546 ERROR (MainThread) [homeassistant.components.light] Platform mqtt does not generate unique IDs. ID ESPlightnurse_led already exists - ignoring light.d1_mqqt2_nurse_led
2025-09-21 11:27:10.558 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.564 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.564 ERROR (MainThread) [homeassistant.components.light] Platform mqtt does not generate unique IDs. ID ESPlightky016_rgb_led already exists - ignoring light.lolinesp32_ky016_rgb_led
2025-09-21 11:27:10.568 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.569 ERROR (MainThread) [homeassistant.components.light] Platform mqtt does not generate unique IDs. ID ESPlightky016_rgb_led already exists - ignoring light.lolinesp32_ky016_rgb_led
2025-09-21 11:27:10.578 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.579 ERROR (MainThread) [homeassistant.components.light] Platform mqtt does not generate unique IDs. ID ESPlightky016_rgb_led already exists - ignoring light.lolinesp32_ky016_rgb_led
2025-09-21 11:27:10.585 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.594 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.611 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.612 ERROR (MainThread) [homeassistant.components.light] Platform mqtt does not generate unique IDs. ID ESPlighttheatre_status_light already exists - ignoring light.theatre_module_theatre_status_light
2025-09-21 11:27:10.623 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.630 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.633 ERROR (MainThread) [homeassistant.components.light] Platform mqtt does not generate unique IDs. ID ESPlighttheatre_status_light already exists - ignoring light.theatre_module_theatre_status_light
2025-09-21 11:27:10.638 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.639 ERROR (MainThread) [homeassistant.components.light] Platform mqtt does not generate unique IDs. ID ESPlighttheatre_status_light already exists - ignoring light.theatre_module_theatre_status_light
2025-09-21 11:27:10.645 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.653 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.660 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.664 ERROR (MainThread) [homeassistant.components.mqtt] The 'color_mode' option has been removed, please remove it from your configuration
2025-09-21 11:27:10.686 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_temperature already exists - ignoring sensor.d1_minipro_room_temperature
2025-09-21 11:27:10.688 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_humidity already exists - ignoring sensor.d1_minipro_room_humidity
2025-09-21 11:27:10.692 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_temperature already exists - ignoring sensor.d1_minipro_room_temperature
2025-09-21 11:27:10.695 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_humidity already exists - ignoring sensor.d1_minipro_room_humidity
2025-09-21 11:27:10.697 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorlast_rfid_tag already exists - ignoring sensor.rfid_last_rfid_tag
2025-09-21 11:27:10.704 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_1_temperature already exists - ignoring sensor.dht22_room_1_temperature
2025-09-21 11:27:10.706 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_1_humidity already exists - ignoring sensor.dht22_room_1_humidity
2025-09-21 11:27:10.717 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorlast_rfid_tag already exists - ignoring sensor.rfid_last_rfid_tag
2025-09-21 11:27:10.718 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_temperature already exists - ignoring sensor.d1_minipro_room_temperature
2025-09-21 11:27:10.724 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_humidity already exists - ignoring sensor.d1_minipro_room_humidity
2025-09-21 11:27:10.731 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorsound_sensor already exists - ignoring sensor.lolinesp32_sound_sensor
2025-09-21 11:27:10.732 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_temperature already exists - ignoring sensor.d1_minipro_room_temperature
2025-09-21 11:27:10.734 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_humidity already exists - ignoring sensor.d1_minipro_room_humidity
2025-09-21 11:27:10.735 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorsound_sensor already exists - ignoring sensor.lolinesp32_sound_sensor
2025-09-21 11:27:10.737 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorlast_rfid_tag already exists - ignoring sensor.rfid_last_rfid_tag
2025-09-21 11:27:10.743 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_temperature already exists - ignoring sensor.d1_minipro_room_temperature
2025-09-21 11:27:10.755 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_humidity already exists - ignoring sensor.d1_minipro_room_humidity
2025-09-21 11:27:10.758 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorsound_sensor already exists - ignoring sensor.lolinesp32_sound_sensor
2025-09-21 11:27:10.765 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorlast_rfid_tag already exists - ignoring sensor.rfid_last_rfid_tag
2025-09-21 11:27:10.776 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorsound_sensor already exists - ignoring sensor.lolinesp32_sound_sensor
2025-09-21 11:27:10.781 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensortheatre_temperature already exists - ignoring sensor.theatre_module_theatre_temperature
2025-09-21 11:27:10.782 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensortheatre_humidity already exists - ignoring sensor.theatre_module_theatre_humidity
2025-09-21 11:27:10.784 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensortheatre_temperature already exists - ignoring sensor.theatre_module_theatre_temperature
2025-09-21 11:27:10.785 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensortheatre_humidity already exists - ignoring sensor.theatre_module_theatre_humidity
2025-09-21 11:27:10.787 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_1_temperature already exists - ignoring sensor.dht22_room_1_temperature
2025-09-21 11:27:10.789 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_1_humidity already exists - ignoring sensor.dht22_room_1_humidity
2025-09-21 11:27:10.791 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_1_temperature already exists - ignoring sensor.dht22_room_1_temperature
2025-09-21 11:27:10.792 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_1_humidity already exists - ignoring sensor.dht22_room_1_humidity
2025-09-21 11:27:10.795 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_1_temperature already exists - ignoring sensor.dht22_room_1_temperature
2025-09-21 11:27:10.796 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_1_humidity already exists - ignoring sensor.dht22_room_1_humidity
2025-09-21 11:27:10.798 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensortheatre_temperature already exists - ignoring sensor.theatre_module_theatre_temperature
2025-09-21 11:27:10.800 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensortheatre_humidity already exists - ignoring sensor.theatre_module_theatre_humidity
2025-09-21 11:27:10.807 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_temperature already exists - ignoring sensor.d1_minipro_room_temperature
2025-09-21 11:27:10.822 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_humidity already exists - ignoring sensor.d1_minipro_room_humidity
2025-09-21 11:27:10.823 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_temperature already exists - ignoring sensor.d1_minipro_room_temperature
2025-09-21 11:27:10.825 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_humidity already exists - ignoring sensor.d1_minipro_room_humidity
2025-09-21 11:27:10.827 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_temperature already exists - ignoring sensor.d1_minipro_room_temperature
2025-09-21 11:27:10.829 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_humidity already exists - ignoring sensor.d1_minipro_room_humidity
2025-09-21 11:27:10.830 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_temperature already exists - ignoring sensor.d1_minipro_room_temperature
2025-09-21 11:27:10.831 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorroom_humidity already exists - ignoring sensor.d1_minipro_room_humidity
2025-09-21 11:27:10.849 ERROR (MainThread) [homeassistant.components.switch] Platform mqtt does not generate unique IDs. ID ESPswitchbuzzer already exists - ignoring switch.lolinesp32_buzzer
2025-09-21 11:27:10.851 ERROR (MainThread) [homeassistant.components.switch] Platform mqtt does not generate unique IDs. ID ESPswitchbuzzer already exists - ignoring switch.lolinesp32_buzzer
2025-09-21 11:27:10.863 ERROR (MainThread) [homeassistant.components.switch] Platform mqtt does not generate unique IDs. ID ESPswitchbuzzer already exists - ignoring switch.lolinesp32_buzzer
2025-09-21 11:27:10.865 ERROR (MainThread) [homeassistant.components.switch] Platform mqtt does not generate unique IDs. ID ESPswitchbuzzer_via_mqtt already exists - ignoring switch.rfid32v2_buzzer_via_mqtt
2025-09-21 11:27:16.391 INFO (MainThread) [homeassistant.components.mqtt.client] MQTT client initialized, birth message sent
2025-09-21 11:27:30.699 WARNING (MainThread) [aioesphomeapi.reconnect_logic] Can't connect to ESPHome API for ward10_bed10_remote @ 192.168.5.168: Error connecting to [AddrInfo(family=<AddressFamily.AF_INET: 2>, type=<SocketKind.SOCK_STREAM: 1>, proto=6, sockaddr=IPv4Sockaddr(address='192.168.5.168', port=6053))]: [Errno 111] Connect call failed ('192.168.5.168', 6053) (SocketAPIError)
2025-09-21 11:34:41.257 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:34:41.258 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:34:41.261 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:34:41.277 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:34:41.286 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:35:06.267 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:35:06.267 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:35:06.277 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:35:06.281 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:35:06.281 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:35:31.255 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:35:31.259 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:35:31.265 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:35:31.267 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:35:31.291 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:35:56.280 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:35:56.303 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:35:56.303 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:35:56.303 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:35:56.323 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:36:21.298 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:36:21.310 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:36:21.312 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:36:21.313 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:36:21.321 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:36:46.315 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:36:46.319 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:36:46.324 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:36:46.330 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:36:46.332 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:37:11.299 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:37:11.313 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:37:11.314 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:37:11.320 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:37:11.340 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:37:36.333 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:37:36.345 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:37:36.361 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:37:36.366 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:37:36.367 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:38:01.346 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:38:01.355 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:38:01.370 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:38:01.373 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:38:01.379 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:38:26.371 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:38:26.374 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:38:26.376 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:38:26.394 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:38:26.394 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:38:51.377 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:38:51.387 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:38:51.391 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:38:51.393 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:38:51.404 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:39:16.391 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:39:16.404 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:39:16.413 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:39:16.418 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:39:16.419 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:39:41.405 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:39:41.417 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:39:41.425 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:39:41.456 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:39:41.456 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:40:06.418 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:40:06.430 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:40:06.431 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:40:06.438 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:40:06.439 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:40:31.429 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:40:31.441 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:40:31.445 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:40:31.450 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:40:31.465 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:40:56.444 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:40:56.454 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:40:56.466 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:40:56.476 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:40:56.481 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:41:21.461 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:41:21.469 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:41:21.476 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:41:21.476 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:41:21.476 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:41:46.468 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:41:46.480 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:41:46.484 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:41:46.487 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:41:46.493 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:42:11.479 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:42:11.492 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:42:11.509 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:42:11.509 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:42:11.511 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:42:36.492 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:42:36.507 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:42:36.508 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:42:36.508 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:42:36.522 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:43:01.510 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:43:01.520 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:43:01.527 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:43:01.529 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:43:01.532 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:43:26.505 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:43:26.517 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:43:26.519 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:43:26.521 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:43:26.544 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:43:51.533 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:43:51.561 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:43:51.562 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:43:51.564 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:43:51.568 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:44:16.549 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:44:16.581 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:44:16.582 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:44:16.582 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:44:16.587 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:44:41.542 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:44:41.579 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:44:41.580 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:44:41.580 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:44:41.583 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:45:06.580 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:45:06.580 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:45:06.596 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:45:06.596 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:45:06.596 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:45:31.582 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:45:31.602 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:45:31.605 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:45:31.609 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:45:31.613 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:45:56.596 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:45:56.611 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:45:56.618 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:45:56.618 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:45:56.629 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:46:21.613 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:46:21.613 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:46:21.622 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:46:21.623 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:46:21.633 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:46:46.620 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:46:46.639 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:46:46.639 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:46:46.640 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:46:46.655 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:47:11.639 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:47:11.647 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:47:11.653 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:47:11.654 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:47:11.656 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:47:36.646 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:47:36.666 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:47:36.668 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:47:36.668 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:47:36.679 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:48:01.664 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:48:01.665 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:48:01.675 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:48:01.675 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:48:01.679 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:48:26.670 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:48:26.680 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:48:26.682 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:48:26.693 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:48:26.698 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:48:51.682 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:48:51.695 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:48:51.695 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:48:51.697 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:48:51.700 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:49:06.182 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt:381]: Can't connect to MQTT... Restarting...
2025-09-21 11:49:25.816 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:49:25.820 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:49:25.834 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:49:25.835 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:49:25.837 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:49:50.807 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:49:50.811 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:49:50.829 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:49:50.831 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:49:50.941 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:50:15.837 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:50:15.865 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:50:15.865 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:50:15.865 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:50:15.865 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:50:40.841 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:50:40.850 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:50:40.850 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:50:40.850 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:50:40.874 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:51:05.844 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:51:05.847 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:51:05.861 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:51:05.864 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:51:05.891 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:51:30.875 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:51:30.887 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:51:30.892 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:51:30.898 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:51:30.898 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:51:55.889 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:51:55.904 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:51:55.904 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:51:55.912 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:51:55.913 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:52:15.886 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:52:15.904 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:52:15.910 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:52:15.910 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:52:15.925 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:52:40.897 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:52:40.905 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:52:40.912 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:52:40.913 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:52:40.941 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:53:05.915 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:53:05.921 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:53:05.922 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:53:05.922 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:53:05.951 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:53:30.942 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:53:30.954 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:53:30.959 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:53:30.961 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:53:30.965 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:53:55.940 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:53:55.965 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:53:55.965 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:53:55.965 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:53:55.977 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:54:20.966 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:54:20.976 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:54:20.987 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:54:20.987 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:54:20.987 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:54:45.975 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:54:45.980 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:54:45.998 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:54:45.999 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:54:45.999 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:55:10.987 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:55:11.004 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:55:11.005 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:55:11.008 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:55:11.014 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:55:35.996 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:55:36.011 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:55:36.012 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:55:36.015 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:55:36.025 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:56:01.011 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:56:01.025 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:56:01.028 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:56:01.028 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:56:01.047 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:56:26.021 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:56:26.044 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:56:26.044 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:56:26.045 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:56:26.052 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:56:51.027 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:56:51.027 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:56:51.034 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:56:51.035 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:56:51.063 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:57:51.164 WARNING (MainThread) [aioesphomeapi.connection] ward10_bed10_remote @ 192.168.5.168: Connection error occurred: ward10_bed10_remote @ 192.168.5.168: EOF received
2025-09-21 11:57:53.703 WARNING (MainThread) [aioesphomeapi.reconnect_logic] Can't connect to ESPHome API for ward10_bed10_remote @ 192.168.5.168: Error connecting to [AddrInfo(family=<AddressFamily.AF_INET: 2>, type=<SocketKind.SOCK_STREAM: 1>, proto=6, sockaddr=IPv4Sockaddr(address='192.168.5.168', port=6053))]: [Errno 111] Connect call failed ('192.168.5.168', 6053) (SocketAPIError)
2025-09-21 11:58:18.105 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:58:18.105 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:58:18.118 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:58:18.118 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:58:18.118 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:58:43.108 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:58:43.124 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:58:43.125 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:58:43.125 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:58:43.148 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:59:08.122 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:59:08.177 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:59:08.177 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:59:08.177 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:59:08.177 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:59:33.133 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:162]: MQTT_EVENT_ERROR
2025-09-21 11:59:33.147 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:164]: Last error code reported from esp-tls: 0x8006
2025-09-21 11:59:33.149 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:165]: Last tls stack error number: 0x0
2025-09-21 11:59:33.150 ERROR (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [E][mqtt.idf:167]: Last captured errno : 0 (Success)
2025-09-21 11:59:33.156 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][mqtt:344]: MQTT Disconnected: TCP disconnected.
2025-09-21 11:59:45.562 WARNING (MainThread) [aioesphomeapi.connection] ward10_bed10_remote @ 192.168.5.168: Connection error occurred: ward10_bed10_remote @ 192.168.5.168: EOF received
2025-09-21 11:59:45.610 WARNING (MainThread) [aioesphomeapi.reconnect_logic] Can't connect to ESPHome API for ward10_bed10_remote @ 192.168.5.168: Error connecting to [AddrInfo(family=<AddressFamily.AF_INET: 2>, type=<SocketKind.SOCK_STREAM: 1>, proto=6, sockaddr=IPv4Sockaddr(address='192.168.5.168', port=6053))]: [Errno 111] Connect call failed ('192.168.5.168', 6053) (SocketAPIError)
2025-09-21 11:59:48.968 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][safe_mode:029]: Last reset occurred too quickly; safe mode will be invoked in 8 restarts
2025-09-21 12:00:18.704 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:257]: Component mqtt took a long time for an operation (68 ms).
2025-09-21 12:00:18.719 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:258]: Components should block for at most 30 ms.
2025-09-21 12:00:41.344 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:257]: Component gpio.binary_sensor took a long time for an operation (98 ms).
2025-09-21 12:00:41.345 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:258]: Components should block for at most 30 ms.
2025-09-21 12:00:44.062 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:257]: Component gpio.binary_sensor took a long time for an operation (634 ms).
2025-09-21 12:00:44.068 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:258]: Components should block for at most 30 ms.
2025-09-21 12:00:46.477 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:257]: Component gpio.binary_sensor took a long time for an operation (98 ms).
2025-09-21 12:00:46.492 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:258]: Components should block for at most 30 ms.
2025-09-21 12:00:46.930 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:257]: Component gpio.binary_sensor took a long time for an operation (92 ms).
2025-09-21 12:00:46.931 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:258]: Components should block for at most 30 ms.
2025-09-21 12:00:47.423 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:257]: Component gpio.binary_sensor took a long time for an operation (93 ms).
2025-09-21 12:00:47.434 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:258]: Components should block for at most 30 ms.
2025-09-21 12:01:40.251 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][api.connection:181]: Home Assistant 2025.7.4 (192.168.5.16) didn't respond to ping request in time. Disconnecting...
2025-09-21 12:02:34.524 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorsound_sensor already exists - ignoring sensor.lolinesp32_sound_sensor
2025-09-21 12:02:49.564 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_temperature, topic: 'bedsidev3_unit/sensor/bedsidev3_room_temperature/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_temperature has device class 'temperature', state class 'measurement' unit '°C' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:02:49.578 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_humidity, topic: 'bedsidev3_unit/sensor/bedsidev3_room_humidity/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_humidity has device class 'humidity', state class 'measurement' unit '%' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:09:01.273 WARNING (MainThread) [aioesphomeapi.connection] ward10_bed10_remote @ 192.168.5.168: Connection error occurred: ward10_bed10_remote @ 192.168.5.168: EOF received
2025-09-21 12:09:22.307 WARNING (MainThread) [aioesphomeapi.reconnect_logic] Can't connect to ESPHome API for ward10_bed10_remote @ 192.168.5.168: Error connecting to [AddrInfo(family=<AddressFamily.AF_INET: 2>, type=<SocketKind.SOCK_STREAM: 1>, proto=6, sockaddr=IPv4Sockaddr(address='192.168.5.168', port=6053))]: [Errno 111] Connect call failed ('192.168.5.168', 6053) (SocketAPIError)
2025-09-21 12:10:29.598 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_temperature, topic: 'bedsidev3_unit/sensor/bedsidev3_room_temperature/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_temperature has device class 'temperature', state class 'measurement' unit '°C' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:10:29.601 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_humidity, topic: 'bedsidev3_unit/sensor/bedsidev3_room_humidity/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_humidity has device class 'humidity', state class 'measurement' unit '%' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:12:24.333 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:257]: Component mqtt took a long time for an operation (67 ms).
2025-09-21 12:12:24.333 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:258]: Components should block for at most 30 ms.
2025-09-21 12:12:26.589 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:257]: Component gpio.binary_sensor took a long time for an operation (93 ms).
2025-09-21 12:12:26.595 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:258]: Components should block for at most 30 ms.
2025-09-21 12:12:27.087 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:257]: Component gpio.binary_sensor took a long time for an operation (88 ms).
2025-09-21 12:12:27.087 WARNING (MainThread) [homeassistant.components.esphome.manager] ward1_bed1_remote: [W][component:258]: Components should block for at most 30 ms.
2025-09-21 12:15:49.668 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorsound_sensor already exists - ignoring sensor.lolinesp32_sound_sensor
2025-09-21 12:15:49.752 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_temperature, topic: 'bedsidev3_unit/status' with payload: b'online'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_temperature has device class 'temperature', state class 'measurement' unit '°C' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:15:49.755 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_humidity, topic: 'bedsidev3_unit/status' with payload: b'online'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_humidity has device class 'humidity', state class 'measurement' unit '%' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:19:05.143 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_temperature, topic: 'bedsidev3_unit/sensor/bedsidev3_room_temperature/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_temperature has device class 'temperature', state class 'measurement' unit '°C' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:19:05.146 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_humidity, topic: 'bedsidev3_unit/sensor/bedsidev3_room_humidity/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_humidity has device class 'humidity', state class 'measurement' unit '%' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:22:25.701 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_temperature, topic: 'bedsidev3_unit/sensor/bedsidev3_room_temperature/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_temperature has device class 'temperature', state class 'measurement' unit '°C' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:22:25.711 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_humidity, topic: 'bedsidev3_unit/sensor/bedsidev3_room_humidity/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_humidity has device class 'humidity', state class 'measurement' unit '%' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:26:35.766 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_temperature, topic: 'bedsidev3_unit/sensor/bedsidev3_room_temperature/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_temperature has device class 'temperature', state class 'measurement' unit '°C' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:26:35.769 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_humidity, topic: 'bedsidev3_unit/sensor/bedsidev3_room_humidity/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_humidity has device class 'humidity', state class 'measurement' unit '%' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:30:57.654 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID ESPsensorsound_sensor already exists - ignoring sensor.lolinesp32_sound_sensor
2025-09-21 12:36:01.911 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_temperature, topic: 'bedsidev3_unit/sensor/bedsidev3_room_temperature/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_temperature has device class 'temperature', state class 'measurement' unit '°C' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:36:01.913 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_humidity, topic: 'bedsidev3_unit/sensor/bedsidev3_room_humidity/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_humidity has device class 'humidity', state class 'measurement' unit '%' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:40:22.057 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_temperature, topic: 'bedsidev3_unit/sensor/bedsidev3_room_temperature/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_temperature has device class 'temperature', state class 'measurement' unit '°C' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:40:22.059 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_humidity, topic: 'bedsidev3_unit/sensor/bedsidev3_room_humidity/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_humidity has device class 'humidity', state class 'measurement' unit '%' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:41:12.025 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_temperature, topic: 'bedsidev3_unit/sensor/bedsidev3_room_temperature/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_temperature has device class 'temperature', state class 'measurement' unit '°C' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
2025-09-21 12:41:12.027 ERROR (MainThread) [homeassistant.components.mqtt.models] Exception raised while updating state of sensor.bedsidev3_unit_bedsidev3_room_humidity, topic: 'bedsidev3_unit/sensor/bedsidev3_room_humidity/state' with payload: b'nan'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/mqtt/models.py", line 366, in process_write_state_requests
    entity.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1018, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1140, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1080, in __async_calculate_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1024, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 679, in state
    raise ValueError(
    ...<5 lines>...
    )
ValueError: Sensor sensor.bedsidev3_unit_bedsidev3_room_humidity has device class 'humidity', state class 'measurement' unit '%' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-finite value: 'nan'
